tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "noEmit": true,
     "isolatedModules": true,
     "strict": true,
-    "noImplicitAny": false,
+    "noImplicitAny": true,
```

Comment:
![giphy (1)](https://user-images.githubusercontent.com/12068482/116831944-bbbd1380-abf5-11eb-9bb4-a6dc0536ab60.gif)

Reply:
How about disabling `any` for the `Field` file which I think is the biggest one ? 
`/* eslint-disable @typescript-eslint/no-explicit-any */`

Reply:
The main thing with this was that I was highlight missing types so that you would have to explicitly have to add ts-ignore or eslint-disable or something along the lines. That way we can trace back through and fix those issues later.